frontend/src/screens/dashboard/docsqa/Applications/index.tsx

```diff
@@ -136,13 +163,38 @@ const Applications = () => {
               {!!applicationData?.questions?.length && (
                 <>
                   <div className="mt-3 text-sm mb-1">Questions:</div>
-                  <ul className='list-disc ml-4'>
-                    {applicationData?.questions?.map((question: string, index: number) => (
-                      <li className='text-sm font-medium' key={index}>{question}</li>
-                    ))}
+                  <ul className="list-disc ml-4">
+                    {applicationData?.questions?.map(
+                      (question: string, index: number) => (
+                        <li className="text-sm font-medium" key={index}>
+                          {question}
+                        </li>
+                      ),
+                    )}
                   </ul>
                 </>
               )}
+
+              <button
+                onClick={() => setShowCurl(!showCurl)}
+                className="my-2 text-sm p-2 bg-[#6366F1] text-white rounded"
+              >
+                {showCurl ? 'Hide' : 'Show Curl Command'}
```

Comment:
i would suggest lets keep 'Curl Command' string for both conditions like :
```suggestion
                {showCurl ? 'Hide' : 'Show'} Curl Command
```

Reply:
I have changes the header, stream option completely, show/hide text, and clipboard icon issue , and made close button fixed in button Review it once @sayan-truefoundry  and ping me if any other fix is needed